### PR TITLE
feat(azure): Add support for TDU purchase model for Azure MSSQL 

### PR DESCRIPTION
**Pull request by ekifel**
+ fix issue #758

### CODE REVIEW

**Comment by ekifel** [comment]
@alikhajeh1 this is ready for review

File: internal/providers/terraform/azure/mssql_database.go [hunk 1]
```diff
@@ -38,6 +38,98 @@ func NewAzureRMMSSQLDatabase(d *schema.ResourceData, u *schema.UsageData) *schem
...
+	skuName := strings.ToLower(sku)
+	if skuName == "basic" {
+		skuName = "b"
+	}
+
+	costComponents = append(costComponents, &schema.CostComponent{
+		Name:            fmt.Sprintf("Compute (%s)", strings.ToTitle(sku)),
+		Unit:            "days",
+		UnitMultiplier:  1,
+		MonthlyQuantity: decimalPtr(decimal.NewFromInt(30)),
```
Comment (alikhajeh1) [inline internal/providers/terraform/azure/mssql_database.go:65]:
```suggestion
                // This is not the same as the 730h/month value we use elsewhere but it looks more understandable than seeing `30.4166` in the output
		MonthlyQuantity: decimalPtr(decimal.NewFromInt(30)),
```

**Review by alikhajeh1 — APPROVED** [review]
LGTM! I don't see a test case changed for the `LicenseIncluded` issue in this PR, is that already covered by other test cases?

**Comment by ekifel** [comment]
@alikhajeh1 yep, I forgot to update golden file after added license logic. Now it covered

**Comment by alikhajeh1** [comment]
Thanks @ekifel!

**Pull request merged**